From d3e167b870e13ee12c7c198762018d72e8f982d1 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sat, 13 Sep 2014 16:55:12 +0200
Subject: [PATCH 17/50] cloud-print: disable in prefs and component loader

Cloud Print is usually autodisabled if no printers are shared.
So what exactly am I turning off?
Probably also needs to gain a checkbox in the settings. Revisit..
---
 chrome/browser/extensions/component_loader.cc | 2 ++
 chrome/browser/ui/browser_ui_prefs.cc         | 2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/chrome/browser/extensions/component_loader.cc b/chrome/browser/extensions/component_loader.cc
index 2362e12..af44282 100644
--- a/chrome/browser/extensions/component_loader.cc
+++ b/chrome/browser/extensions/component_loader.cc
@@ -488,10 +488,12 @@ void ComponentLoader::AddDefaultComponentExtensions(
   DCHECK(!skip_session_components);
   Add(IDR_BOOKMARKS_MANIFEST,
       base::FilePath(FILE_PATH_LITERAL("bookmark_manager")));
+#if 0
   // Cloud Print component app. Not required on Chrome OS.
   Add(IDR_CLOUDPRINT_MANIFEST,
       base::FilePath(FILE_PATH_LITERAL("cloud_print")));
 #endif
+#endif
 
   if (!skip_session_components) {
     AddWebStoreApp();
diff --git a/chrome/browser/ui/browser_ui_prefs.cc b/chrome/browser/ui/browser_ui_prefs.cc
index 20848f0..5c70c06 100644
--- a/chrome/browser/ui/browser_ui_prefs.cc
+++ b/chrome/browser/ui/browser_ui_prefs.cc
@@ -114,7 +114,7 @@ void RegisterBrowserUserPrefs(user_prefs::PrefRegistrySyncable* registry) {
       user_prefs::PrefRegistrySyncable::UNSYNCABLE_PREF);
   registry->RegisterBooleanPref(
       prefs::kCloudPrintSubmitEnabled,
-      true,
+      false,
       user_prefs::PrefRegistrySyncable::UNSYNCABLE_PREF);
   registry->RegisterBooleanPref(
       prefs::kDevToolsDisabled,
-- 
2.4.3

